clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -348,11 +356,17 @@ def rm(self, path, recursive=False, maxdepth=None):
         )
         actual_path = context_pair.actual_file_location()
         storage_type = self._recognize_storage_type(actual_path)
-        context_pair.filesystem().rm(
-            self._strip_storage_protocol(storage_type, actual_path),
-            recursive,
-            maxdepth,
-        )
+        fs = context_pair.filesystem()
+
+        # S3FileSystem doesn't support maxdepth
+        if isinstance(fs, self.lazy_load_class("s3fs", "S3FileSystem")):
```

Comment:
In the latest doc, s3fs seems supports the max depth param(https://s3fs.readthedocs.io/en/latest/api.html#s3fs.core.S3FileSystem.rm), is there something wrong?

Reply:
It actually not and the error will be:

```
======================================================================
ERROR: test_rm (test_gvfs_with_s3.TestGvfsWithS3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ec2-user/gravitino/clients/client-python/tests/integration/test_gvfs_with_s3.py", line 195, in test_rm
    fs.rm(rm_file)
  File "/home/ec2-user/gravitino/clients/client-python/gravitino/filesystem/gvfs.py", line 355, in rm
    context_pair.filesystem().rm(
  File "/home/ec2-user/gravitino/.gradle/python/Linux/Miniforge3/envs/python-3.8/lib/python3.8/site-packages/fsspec/asyn.py", line 118, in wrapper
    return sync(self.loop, func, *args, **kwargs)
  File "/home/ec2-user/gravitino/.gradle/python/Linux/Miniforge3/envs/python-3.8/lib/python3.8/site-packages/fsspec/asyn.py", line 85, in sync
    coro = func(*args, **kwargs)
TypeError: _rm() takes from 2 to 3 positional arguments but 4 were given

```

Reply:
I see. I checked the s3fs code, it does not support max depth indeed: https://github.com/fsspec/s3fs/blob/main/s3fs/core.py#L2001.